lib/log.rb

```diff
@@ -115,7 +115,7 @@ def log(msg, level=INFO, show_datetime='time')
         timestamp = ''
       end
 
-      m = {msg: msg, level: level, timestamp: timestamp}
+      m = {msg: censor(msg), level: level, timestamp: timestamp}
```

Comment:
A few lines below, we have `print(construct(m))`, m calls censor, and in the print method, we calls again censor, is it dup ?

Reply:
Nice catch!

Reply:
Fixed